spec/confirmation_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'confirmation'
+
+describe Confirmation do
+	checkout = {fish_and_chips: [5.99,2]}
+	subject(:confirmation) {described_class.new(checkout)}
+	let(:our_total) {11.98}
+	let(:user_total) {15}
+
+	context '#initialize' do
+		it 'assigns checkout to a variable' do
+			expect(confirmation.checkout).to eq checkout
+		end
+		it 'creates a variable to store the total' do
+			expect(confirmation.total).to eq 0
+		end
+	end
+
+	context '#user_cost' do
+		it "should raise an error if totals don't match" do
+			expect{confirmation(confirmation.user_cost(user_total))}.to raise_error "The totals don't match, ours: £#{our_total} yours: £#{user_total}!"
```

Comment:
Metrics/LineLength: Line is too long. [143/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)